app/cdap/components/WrangleHome/Components/OngoingDataExplorations/types.ts

```diff
@@ -78,3 +80,30 @@ export interface IMassagedObject {
   subText: string;
   workspaceId: string;
 }
+
+export interface IEachData {
```

Comment:
rename IEachData to more appropriate one
e.g. IExplorationCard

Reply:
DOne